src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -21,6 +24,15 @@
     @NotBlank
     private String contentText;
 
+    @ManyToOne
+    @JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "name")
+    @JsonIdentityReference(alwaysAsId = true)
+    @JoinColumn(nullable = false)
+    private User relatedUser;
+    
+    @OneToMany(mappedBy = "likedPost", cascade = CascadeType.ALL)
+    private List<PostLike> listOfCommentLikes;
```

Comment:
Were we not going to have a separate CommentLike model?

Reply:
Agree, PostLike can only have a list of Post type. It cannot store Comment type.

Reply:
Yes we will have seperate CommentLike model when we implement the likes for comment